docs/standard/serialization/system-text-json/polymorphism.md

```diff
@@ -136,7 +139,607 @@ The following example shows the JSON that results from the preceding code:
 ```
 
 > [!NOTE]
-> This article is about serialization, not deserialization. Polymorphic deserialization is not supported, but as a workaround you can write a custom converter, such as the example in [Support polymorphic deserialization](converters-how-to.md#support-polymorphic-deserialization).
+> This article is about serialization, not deserialization. Polymorphic deserialization is not supported, but as a workaround you can write a custom converter, such as the example in [Support polymorphic deserialization](converters-how-to.md#support-polymorphic-deserialization). .NET 7 supports polymorphic serialization and deserialization, for more information see [How to serialize properties of derived classes with System.Text.Json in .NET 7](polymorphism.md?pivots=dotnet-7-0).
+
+:::zone-end
+:::zone pivot="dotnet-7-0"
+
+[!INCLUDE [preview-content](../../../../includes/preview-content.md)]
```

Comment:
This states "This content relies on NuGet packages..." - but in this case you don't necessarily need a NuGet package.

Reply:
Good catch, I'll remove that altogether. The zone pivot already says **.NET 7 Preview**.